airbyte-cdk/python/airbyte_cdk/sources/declarative/schema/default_schema_loader.py

```diff
@@ -5,15 +5,16 @@
 from dataclasses import InitVar, dataclass
 from typing import Any, Mapping
 
+from airbyte_cdk.sources.declarative.schema.json_file_schema_loader import JsonFileSchemaLoader
 from airbyte_cdk.sources.declarative.schema.schema_loader import SchemaLoader
 from airbyte_cdk.sources.declarative.types import Config
 from dataclasses_jsonschema import JsonSchemaMixin
 
 
 @dataclass
-class EmptySchemaLoader(SchemaLoader, JsonSchemaMixin):
+class DefaultSchemaLoader(SchemaLoader, JsonSchemaMixin):
```

Comment:
The alternative here would have been to fold the defaulting to the empty schema logic into the existing JsonFileSchemaLoader and have that remain as the default. The drawback there is that it starts to blur the line between how errors should be reported when schema files don't exist. It might be good to have very clear failures when a developer explicitly defines their SchemaLoader like when a file does not exist. However, when using the default we should just continue w/ the empty schema like we do for this component.